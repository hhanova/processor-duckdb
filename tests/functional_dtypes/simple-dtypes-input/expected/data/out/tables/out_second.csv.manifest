{"destination": "out_second.csv", "incremental": true, "write_always": false, "delimiter": ",", "enclosure": "\"", "manifest_type": "out", "has_header": false, "schema": [{"name": "STRING", "data_type": {"base": {"type": "STRING"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "STRING", "KBC.datatype.nullable": false}}, {"name": "INTEGER", "data_type": {"base": {"type": "INTEGER"}}, "nullable": true, "primary_key": true, "metadata": {"KBC.datatype.basetype": "INTEGER", "KBC.datatype.nullable": false}}, {"name": "NUMERIC", "data_type": {"base": {"type": "FLOAT"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "FLOAT", "KBC.datatype.nullable": false}}, {"name": "FLOAT", "data_type": {"base": {"type": "FLOAT"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "FLOAT", "KBC.datatype.nullable": false}}, {"name": "BOOLEAN", "data_type": {"base": {"type": "BOOLEAN"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "BOOLEAN", "KBC.datatype.nullable": false}}, {"name": "DATE", "data_type": {"base": {"type": "DATE"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "DATE", "KBC.datatype.nullable": false}}, {"name": "TIMESTAMP", "data_type": {"base": {"type": "TIMESTAMP"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "TIMESTAMP", "KBC.datatype.nullable": false}}, {"name": "/table.parquet", "data_type": {"base": {"type": "STRING"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "STRING", "KBC.datatype.nullable": false}}, {"name": "new_column", "data_type": {"base": {"type": "INTEGER"}}, "nullable": true, "metadata": {"KBC.datatype.basetype": "INTEGER", "KBC.datatype.nullable": false}}]}