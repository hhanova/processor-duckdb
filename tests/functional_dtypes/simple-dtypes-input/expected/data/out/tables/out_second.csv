STRING,INTEGER,NUMERIC,FLOAT,BOOLEAN,DATE,TIMESTAMP,/table.parquet,new_column
"two","2","2.2","2.22","false","2023-02-01","2023-02-01 12:00:00","/table.parquet","2"
"three","3","3.3","3.33","true","2023-03-01","2023-03-01 12:00:00","/table.parquet","2"
"four","4","4.4","4.44","false","2023-04-01","2023-04-01 12:00:00","/table.parquet","2"
"five","5","5.5","5.55","true","2023-05-01","2023-05-01 12:00:00","/table.parquet","2"
"six","6","6.1","6.11","true","2023-01-01","2023-01-01 12:00:00","/table2.parquet","2"
"seven","7","7.2","7.22","false","2023-02-01","2023-02-01 12:00:00","/table2.parquet","2"
"eight","8","8.3","8.33","true","2023-03-01","2023-03-01 12:00:00","/table2.parquet","2"
"nine","9","9.4","9.44","false","2023-04-01","2023-04-01 12:00:00","/table2.parquet","2"
"ten","10","10.5","10.55","true","2023-05-01","2023-05-01 12:00:00","/table2.parquet","2"
